packages/lex-cli/src/codegen/server.ts

```diff
@@ -446,6 +446,10 @@ function genServerXrpcMethod(
         type: 'Uint8Array | stream.Readable',
       })
     }
+    handlerSuccess.addProperty({
+      name: 'headers?',
+      type: '{ [key: string]: string }',
```

Comment:
updated the `Record` type here because it was conflicting with the generated `Record` interface